adafruit_ht16k33/segments.py

```diff
@@ -211,21 +215,62 @@ def _text(self, text):
         for character in text:
             self._push(character)
 
-    def _number(self, number):
-        """Display the specified decimal number."""
+    '''
+		Display a floating point or integer number on the Adafruit HT16K33 based displays
+		
+		Param: number - The floating point or integer number to be displayed, which must be
+			in the range 0 (zero) to 9999 for integers and floating point or integer numbers
+			and between 0.0 and 999.0 or 99.00 or 9.000 for floating point numbers.
+		Param: decimal - The number of decimal places for a floating point number if decimal
+			is greater than zero, or the input number is an integer if decimal is zero.
+
+        Returns: The output text string to be displayed.'''
```

Comment:
This docstring should be moved into the correct function.

Reply:
Done